decidim-proposals/app/presenters/decidim/proposals/proposal_presenter.rb

```diff
@@ -34,24 +34,14 @@ def display_mention
         link_to title, proposal_path
       end
 
-      def title
+      def title(links: false, extras: true)
```

Comment:
Param docs?

Reply:
Also, about the defaults, is it more common or expected to have these as true or false?

Reply:
Yes, it is much common to use params as given (don't render hashtag links, but include extras). That's why I leave it that way